src/pages/patientView/PatientViewPage.tsx

```diff
@@ -205,7 +201,7 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
 
         if (patientViewPageStore.studyMetaData.isComplete) {
             let study = patientViewPageStore.studyMetaData.result;
-            studyName = <a href={`study?id=${study.studyId}`} className="studyMetaBar_studyName">{study.name}</a>;
+            studyName = <Link to={`/newstudy?id=${study.studyId}`} className="studyMetaBar_studyName">{study.name}</Link>;
```

Comment:
why not /study? is /newstudy ready? and when it is, arent we using study?

Reply:
i don't think it's been decided yet.  but you're right.   i will make ticket to remove "newstudy"